services/manage/app/controllers/manage/orders/edit/lineitems/edit/ads/edit.js

```diff
@@ -88,7 +96,29 @@ export default Controller.extend(ActionMixin, ObjectQueryManager, {
         this.endAction();
       }
     },
+    /**
+     *
+     */
+    async pause() {
+      this.startAction();
+      this.set('isPausing', true);
+      const id = this.get('model.id');
+      const value = !this.get('model.paused');
+      const variables = { input: { id, value } };
+      const mutation = pauseAd;
+      try {
+        await this.get('apollo').mutate({ mutation, variables }, 'pauseAd');
+      } catch (e) {
+        this.get('graphErrors').show(e);
```

Comment:
This should throw the error so the form component can handle it as a validation error message